src/App.css

```
@@ -138,32 +165,47 @@
-	background: #001027;
+	background: rgba(255, 255, 255, 0.04);
 }
 
 .secondary-title {
 	font-family: 'Montserrat';
-	font-size: 2.4rem;
+	font-size: 1.25rem;
 	text-align: left;
 	font-weight: 600;
 
 	line-height: 1.2;
```

Stylelint (17.2.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)